modules/net-vpc-peering/variables.tf

```diff
@@ -19,6 +19,12 @@ variable "local_network" {
   type        = string
 }
 
+variable "override_name" {
```

Comment:
This module handles up to two peerings, so I'd make this variable a object with two keys (`local` and `peer`, just like `routes_config`) with `optional(string)` values.

Also, I'd simply call it `name`